### ISSUE TITLE
FE: Build Federal Footer in vets-website

### ISSUE THREAD

**Issue by randimays** [department-of-veterans-affairs/va.gov-cms#16809]
## Description

Ahead of the final rounds of Federal Header/Footer research, we are beginning the build of the new designs in vets-website. This ticket is specifically for the **Footer**, and includes accessibility changes indicated in the CodePen prototype code.

[Full Codepen Prototype](https://codepen.io/Sitewide-Team/live/eYxooWQ/0628a3dc13a4678d0a8249c64342f634) for reference

Figma designs are below. **These mockups are the finalized designs** and may vary slightly from the CodePen prototype used in research.
- [Desktop](https://www.figma.com/file/l35ykhDl0U885wSbmrc7Gc/USA-header-footer-VA?type=design&node-id=0%3A2&mode=design&t=e7tv1g0fDiXsCnFL-1)
- [Mobile](https://www.figma.com/file/l35ykhDl0U885wSbmrc7Gc/USA-header-footer-VA?type=design&node-id=456%3A5970&mode=design&t=1weD9RILp4mAeAZx-1)

### User story

**AS A** AS A Veteran, their family member, caregiver, or survivor
**I WANT** to be able to easily understand and navigate the VA.gov footer
**SO THAT** I'm not frustrated, and can find what I need in the least amount of time

### Engineering notes / background

In order to keep any of this in-progress code from being accessible/visible in production, we can create a long-living Tugboat instance targeting the WIP vets-website branch.

### Quality / testing notes
- Unit and e2e tests must be adjusted to fit any new footer HTML structure or dynamic data
- Testing the footer on mobile with this WIP branch will require running vets-website locally, spinning up an ngrok server, and using the ngrok link to review. Tugboat does not work on mobile devices (for lack of SOCKS).

## Acceptance criteria
Definition of "done" for this ticket: a working Tugboat with the WIP footer. **No deploys to staging or production**:exclamation:

- [ ] USA Logo should be behind a flipper, or some type of feature flag so we can do a content toggle between new USA branding and the alternate presentation of the US government text. (blocked by verifying we have designs from Amanda).
- [ ] Injected footer is not updated with new designs
 - [ ] Injected footer is not regressed
- [ ] Desktop footer created with all footer data from Drupal's new menus (new ticket TBD)
- [ ] Markup should be match CodePen as much as possible, including accessibility changes
- [ ] Mobile footer created with all footer data from Drupal new menus (new ticket TBD)
- [ ] Accessibility review
- [ ] Design review by Amanda K
- [ ] Design review by Dave C

**Comment by aklausmeier** [comment]
@randimays updated Figma links to correct frame in files

**Comment by randimays** [comment]
Per Dave/Fran:

> This project is placed on hold... the OFCIO wants to extend the research/usability study across a broader spectrum than only VA.gov. Undetermined at this time which team or who will do the actual work. Moving to Blocked.

**Comment by FranECross** [comment]
Per Dave Conlon, the Federal Header/Footer improvement effort has been closed by the Whitehouse / 18f. All related tickets may be closed.
